circle.yml

```diff
@@ -0,0 +1,9 @@
+test:
+  override:
+    - "true"
+deployment:
+  master:
+    branch: master
+    commands:
+      - npm install
+      - node publish.js
```

Comment:
This should be wrapped in an [NPM script](https://docs.npmjs.com/cli/run-script).